azure/nat-gateway/terraform.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.83.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.1.0"
+    }
```

Comment:
This can be deleted since it's not being used anywhere.

Reply:
removed